### PR TITLE
contrib/dopewars: new package (1.6.2)

### PR DESCRIPTION
**Pull request by mikoxyz**

### CODE REVIEW

**Comment by nekopsykose** [comment]
what

File: contrib/dopewars-console/files/dopewars [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# dopewars service
+
+type        = bgprocess
+command     = /usr/bin/dopewars -S -r /run/dopewars/dopewars.pid
+run-as      = _dopewars
+pid-file    = /run/dopewars/dopewars.pid
+depends-on  = local.target
+after       = network.target
```
Comment (q66) [inline contrib/dopewars-console/files/dopewars:8]: targets are always depends-on

File: contrib/dopewars-console/files/dopewars [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# dopewars service
+
+type        = bgprocess
```
Comment (q66) [inline contrib/dopewars-console/files/dopewars:3]: figure out how to foreground it, we don't take bgprocess services in general

File: contrib/dopewars-console/template.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+pkgname = "dopewars-console"
```
Comment (q66) [inline contrib/dopewars-console/template.py:1]: what is the purpose of a separate console package

**Pull request closed without merging**